A3A/addons/core/functions/Revive/fn_handleDamage.sqf

```diff
@@ -124,11 +123,27 @@ else
 			{
 				if !(_unit getVariable ["incapacitated",false]) then
```

Comment:
What's going on with this condition? Will incapacitated not always be true since it's set up there at line 47 or am I missing something?

Reply:
Also, I don't think going getting downed should have a delay if your one time revive is already used, it looks like you're always gonna have the delay, but the self revive will only work once, this should be reworked to only having the delay if you're gonna be able to self revive, I think.

Although I could be reading it wrong, the formatting is omega fucked on the review page here.

Reply:
It should be working as intended now.